test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs

```diff
@@ -46,7 +46,7 @@ public class CustomDbContext<TKey> : DbContext where TKey : IEquatable<TKey>
 
         [TestPriority(-1000)]
         [ConditionalFact]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
+        [FrameworkSkipCondition(RuntimeFrameworks.Mono)][OSSkipCondition(OperatingSystems.Linux)][OSSkipCondition(OperatingSystems.MacOSX)]
```

Comment:
These are one off, standalone non shared tests, so they can continue to use the skip attributes